template/ios/HelloWorld/AppDelegate.mm

```diff
@@ -54,6 +54,12 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   rootViewController.view = rootView;
   self.window.rootViewController = rootViewController;
   [self.window makeKeyAndVisible];
+
+  CADisplayLink *displayLink = [CADisplayLink displayLinkWithTarget:self selector:@selector(displayLinkTriggered)];
```

Comment:
Could you clarify why it is needed to create a new CADDisplayLink here? 
Isn't the displayLink destroyed at the end of the scope? 

Where is selector `displayLinkTriggered` defined?